src/main/java/org/broadinstitute/hellbender/tools/gvs/filtering/ExtractFeaturesEngine.java

```diff
@@ -133,16 +133,26 @@ public void traverse() {
         final String userDefinedFunctions = ExtractFeaturesBQ.getVQSRFeatureExtractUserDefinedFunctionsString();
         Map<String, String> cleanQueryLabels = createQueryLabels(queryLabels);
 
-        final StorageAPIAvroReader storageAPIAvroReader = BigQueryUtils.executeQueryWithStorageAPI(
+        final StorageAPIAvroReaderAndBigQueryStatistics storageAPIAvroReaderAndBigQueryStatistics = BigQueryUtils.executeQueryWithStorageAPI(
                 featureQueryString,
                 SchemaUtils.FEATURE_EXTRACT_FIELDS,
                 projectID,
                 datasetID,
                 userDefinedFunctions,
                 useBatchQueries,
                 cleanQueryLabels);
+        this.bigQueryQueryByteScanned = storageAPIAvroReaderAndBigQueryStatistics.queryStatistics.getTotalBytesProcessed();
+        this.storageAPIByteScanned = storageAPIAvroReaderAndBigQueryStatistics.storageAPIAvroReader.getEstimatedTotalBytesScanned();
```

Comment:
nit: bytes (plural) scanned for both of these variable names

Reply:
👍